openmetadata-ui/src/main/resources/ui/cypress/common/Utils/Owner.ts

```diff
@@ -69,12 +69,20 @@ export const addOwner = (
   dataTestId?: string,
   verifyPatchResponse = true
 ) => {
+  interceptURL(
+    'GET',
+    '/api/v1/search/query?q=*&index=team_search_index*',
+    'getTeams'
+  );
   interceptURL('GET', '/api/v1/users?*isBot=false*', 'getUsers');
   cy.get('[data-testid="edit-owner"]')
     .scrollIntoView()
     .click({ waitForAnimations: false });
 
   cy.get("[data-testid='select-owner-tabs']").should('be.visible');
+
+  verifyResponseStatusCode('@getTeams', 200); // wait for teams to load before switching the tab
```

Comment:
This won't work in case of update since in update if owner is already set as User, teams API won't fire

Reply:
The teams API get fire too at the time of update user, as it fetches team count at that time.

<img width="1440" alt="image" src="https://github.com/open-metadata/OpenMetadata/assets/66266464/0cfdbaa8-5ee2-465f-bcc9-3ab6a5221180">